core/integration/features/price_monitoring/3362-failing-system-tests.feature

```diff
@@ -56,36 +56,28 @@ Feature: Replicate failing system tests after changes to price monitoring (not t
       | 100000     | 100150            | TRADING_MODE_CONTINUOUS |
 
     When the parties place the following orders:
-      | party  | market id | side | volume | price  | resulting trades | type       | tif     |
-      | party1 | ETH/DEC20 | buy  | 1      | 99845  | 0                | TYPE_LIMIT | TIF_GTC |
-      | party1 | ETH/DEC20 | buy  | 2      | 99844  | 0                | TYPE_LIMIT | TIF_GTC |
+      | party  | market id | side | volume | price | resulting trades | type       | tif     |
+      | party1 | ETH/DEC20 | buy  | 1      | 99845 | 0                | TYPE_LIMIT | TIF_GTC |
+      | party1 | ETH/DEC20 | buy  | 2      | 99844 | 0                | TYPE_LIMIT | TIF_GTC |
     Then the trading mode should be "TRADING_MODE_CONTINUOUS" for the market "ETH/DEC20"
     # Now place a FOK order that would trigger a price auction (party 1 has a buy at 95,000 on the book
 
     And the order book should have the following volumes for market "ETH/DEC20":
       | side | price  | volume |
-      | sell | 107000 | 306    |
-      | buy  | 99845  | 321    |
+      | sell | 107010 | 150    |
+      | sell | 107000 | 6      |
+      | buy  | 99845  | 1      |
       | buy  | 99844  | 2      |
+      | buy  | 99835  | 152    |
       | buy  | 95000  | 6      |
 
     And the parties place the following orders:
-      | party  | market id | side | volume | price | resulting trades | type        | tif     | error                                                       |
-      | party2 | ETH/DEC20 | sell | 322    | 0     | 0                | TYPE_MARKET | TIF_FOK | OrderError: non-persistent order trades out of price bounds |
+      | party  | market id | side | volume | price  | resulting trades | type       | tif     |
+      | party2 | ETH/DEC20 | sell | 322    | 106000 | 0                | TYPE_LIMIT | TIF_GTC |
 
     Then the market data for the market "ETH/DEC20" should be:
       | mark price | last traded price | trading mode            | horizon | min bound | max bound |
       | 100000     | 100150            | TRADING_MODE_CONTINUOUS | 5       | 99845     | 100156    |
       | 100000     | 100150            | TRADING_MODE_CONTINUOUS | 10      | 99711     | 100290    |
 
-    ## Now place the order for the same volume again, but set price to 100,000 -> the buy at 95,000 doesn't uncross
```

Comment:
I think removing these means that the test is no longer testing what it was.

Reply:
LP bond has been reduced, hence the party did not get close-out, and then close-out happens when the book has enough volume